<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_projection_PKAEMO4</class>
 <widget class="QMainWindow" name="Window_projection_PKAEMO4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Проекция (ПК АЭМО 4)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resourse.qrc">
    <normaloff>:/resourse/default/layer_stack_arrange.png</normaloff>:/resourse/default/layer_stack_arrange.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Load">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Загрузить</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/resourse/default/fileopen.png</normaloff>:/resourse/default/fileopen.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(240, 240, 240)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTableWidget" name="tableWidget_files">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Путь</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Пуск</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/resourse/default/player_play.png</normaloff>:/resourse/default/player_play.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Сохранить в:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_Clear">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/resourse/default/edit_clear_list.png</normaloff>:/resourse/default/edit_clear_list.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="toolButton_FlePath">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
  <include location="../../resourse.qrc"/>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
